src/SWP/TemplateEngineBundle/Model/Container.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Template Engine Bundle.
+ *
+ * Copyright 2015 Sourcefabric z.u. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2015 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+
+namespace SWP\TemplateEngineBundle\Model;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use SWP\TemplatesSystem\Gimme\Model\ContainerInterface;
+
+/**
+ * Container.
+ */
+class Container implements ContainerInterface
+{
+    const TYPE_SIMPLE = 1;
+
+    /**
+     * @var int
```

Comment:
you are implementing the `ContainerInterface` the doc block for each property should be like:

```
/**
 * {@inheritdoc}
 */
```

we should keep an eye on it becuase we will need to change doc blocks in each class when we will implement some  documenation generator (e.g. https://github.com/FriendsOfPHP/Sami)